src/internal/pjs/server_test.go

```diff
@@ -0,0 +1,105 @@
+package pjs
+
+import (
+	"context"
+	"testing"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/clusterstate"
+	"github.com/pachyderm/pachyderm/v2/src/internal/dockertestenv"
+	"github.com/pachyderm/pachyderm/v2/src/internal/migrations"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
+	"github.com/pachyderm/pachyderm/v2/src/internal/require"
+	storagesrv "github.com/pachyderm/pachyderm/v2/src/internal/storage"
+	"github.com/pachyderm/pachyderm/v2/src/internal/testetcd"
+	"github.com/pachyderm/pachyderm/v2/src/pjs"
+	"github.com/pachyderm/pachyderm/v2/src/storage"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+)
+
+func TestCreateJob(t *testing.T) {
+	t.Run("valid/parent/nil", func(t *testing.T) {
+		c, fc := setupTest(t)
+		resp := createJob(t, c, fc)
+		t.Log(resp)
+	})
+}
+
+func TestInspectJob(t *testing.T) {
+	t.Run("valid", func(t *testing.T) {
+		c, fc := setupTest(t)
+		ctx := pctx.TestContext(t)
+		programFileset := createFileSet(t, fc, map[string][]byte{
+			"file/path": []byte(`!#/bin/bash; ls /input/;`),
+		})
+		inputFileset := createFileSet(t, fc, map[string][]byte{
+			"a.txt": []byte(`dummy input`),
+		})
```

Comment:
InspectJob should error if it is called before the first create.  Can we add a case for that?

We should return an error with the not found code.
https://pkg.go.dev/google.golang.org/grpc/codes#Code